bftengine/tests/SigManager/SigManager_test.cpp

```diff
@@ -105,9 +105,10 @@ TEST(RsaSignerAndRsaVerifierTest, LoadSignVerifyFromPemfiles) {
   ASSERT_FALSE(verifier_->verify(data1, RANDOM_DATA_SIZE, sig.data(), expectedVerifierSigLen));
 
   // change signature randomally, expect failure
-  string sig1(sig);
-  corrupt(sig1.data(), 1);
-  ASSERT_FALSE(verifier_->verify(data, RANDOM_DATA_SIZE, sig1.data(), expectedVerifierSigLen));
+  char sig1[RANDOM_DATA_SIZE] = {'\0'};
+  strncpy(sig1, sig.c_str(), sig.length());
```

Comment:
The original code copies the content of `sig` to `sig1`, why do you use low-level C-functions?
Applies to all the changes below.

Reply:
@f-squirrel The intension of using low-level C-style string copy was to allocate a character array with enough size (so that no overflow occurs). This (C-style copy) could have been also worked, however it is now replaced by using allocation of memory block by string `reserve()` api.
Resolved here https://github.com/vmware/concord-bft/pull/1750/commits/b23db40f5748c5f5058e445bdfaf2c47a5532283